ovirt/stubs/imagetransfers/stub.json

```diff
@@ -24,7 +24,7 @@
     "method": "POST",
     "responses": [
       {
-        "responseBody": "<image_transfer href=\"/ovirt-engine/api/imagetransfers/abc123\" id=\"abc123\"><direction>download</direction><format>raw</format><phase>transferring</phase><transfer_url>https://fakeovirt.konveyor-forklift:12345/images/cirros</transfer_url></image_transfer>",
+        "responseBody": "<image_transfer href=\"/ovirt-engine/api/imagetransfers/abc123\" id=\"abc123\"><direction>download</direction><format>raw</format><phase>transferring</phase><transfer_url>http://fakeovirt.konveyor-forklift:12345/images/cirros</transfer_url></image_transfer>",
```

Comment:
Should this be port 80? If you prefer to keep 12345, you can expose that port in the `Dockerfile` and the `fakeovirt_deployment.yml`, but whatever you pick, it needs to be consistent.

Reply:
What I understand the imageio server serves the images on port 80 so we need to map it to the 12345

Reply:
Maybe I misunderstood the line. The containerized server exposes port 80 to serve the image, yes. So where the `12345` comes from in this? This is not trying to communicate with the imageio server?

Reply:
It's from the service
```
  - name: imageio
    port: 12345
    targetPort: 80
```